glmtools/glm4d.py

```diff
@@ -54,4 +68,24 @@ def t_test_3d(c, X, B, sigma_2, df):
         two-tailed probability value for each t statistic.
     """
     # Your code code here
-    return
+
+    B_2d = B.reshape((-1, B.shape[-1]))
+
```

Comment:
Consider dropping the spaces between lines here.  General Python coding advice is to use new lines sparingly - see : https://www.python.org/dev/peps/pep-0008/#blank-lines.  Also see https://bic-berkeley.github.io/psych-214-fall-2016/coding_style.html for some general advice.